### ISSUE TITLE
TypeError: undefined is not an object (evaluating 'parent.parent.console.warn')

### ISSUE THREAD

**Issue by JohnMcLear** [ether/etherpad-lite#2429]
```
[2015-01-02 18:27:48.335] [INFO] access - [LEAVE] Pad "release-150": Author "a.zbkpqqjYyzYE61cY" on client wwGS8f96KPObJRT7AADy with IP  left the pad
[2015-01-02 18:27:48.424] [INFO] ueberDB - Flushed 1 values
[2015-01-02 18:27:48.530] [INFO] ueberDB - Flushed 3 values
[2015-01-02 18:27:48.606] [WARN] client - Uncaught Error: doRepApplyChangeset length mismatch: 454/559 -- { errorId: 'fOTv77TFxTNJbNwLHd1u',
  msg: 'Uncaught Error: doRepApplyChangeset length mismatch: 454/559',
  url: 'http://beta.etherpad.org/p/release-150',
  linenumber: 482,
  userAgent: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/38.0.2125.111 Safari/537.36' }
[2015-01-02 18:27:48.631] [INFO] ueberDB - Flushed 2 values
[2015-01-02 18:27:48.702] [WARN] client - TypeError: undefined is not an object (evaluating 'parent.parent.console.warn') -- { errorId: 'UUjmQQb1ultTbvrwu1LS',
  msg: 'TypeError: undefined is not an object (evaluating \'parent.parent.console.warn\')',
  url: 'http://beta.etherpad.org/p/release-150',
  linenumber: 3121,
  userAgent: 'Mozilla/5.0 (iPad; CPU OS 8_1_2 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) Mobile/12B440' }
[2015-01-02 18:27:48.733] [INFO] ueberDB - Flushed 2 values
[2015-01-02 18:27:48.875] [WARN] client - TypeError: undefined is not an object (evaluating 'parent.parent.console.warn') -- { errorId: 'dUSZbDNDwaSNs4NvGb9A',
  msg: 'TypeError: undefined is not an object (evaluating \'parent.parent.console.warn\')',
  url: 'http://beta.etherpad.org/p/release-150',
  linenumber: 3121,
  userAgent: 'Mozilla/5.0 (iPad; CPU OS 8_1_2 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) Mobile/12B440' }
[2015-01-02 18:27:48.933] [INFO] ueberDB - Flushed 1 values
[2015-01-02 18:27:49.040] [INFO] ueberDB - Flushed 3 values
[2015-01-02 18:27:49.082] [WARN] client - Uncaught Error: doRepApplyChangeset length mismatch: 455/559 -- { errorId: 'mb6nqP1vCAStQ60OQwcE',
  msg: 'Uncaught Error: doRepApplyChangeset length mismatch: 455/559',
  url: 'http://beta.etherpad.org/p/release-150',
  linenumber: 482,
  userAgent: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/38.0.2125.111 Safari/537.36' }
[2015-01-02 18:27:49.143] [INFO] ueberDB - Flushed 2 values
[2015-01-02 18:27:49.190] [WARN] client - TypeError: undefined is not an object (evaluating 'parent.parent.console.warn') -- { errorId: 'GsV159ods9TMaSqBrcM5',
  msg: 'TypeError: undefined is not an object (evaluating \'parent.parent.console.warn\')',
  url: 'http://beta.etherpad.org/p/release-150',
  linenumber: 3121,
```

**Comment by JohnMcLear** [comment]
appears to only be an issue when embedded

**Comment by JohnMcLear** [comment]
cc @luto

**Comment by JohnMcLear** [comment]
Closing this in favor of the debug available at #2634
